internal/server/server.go

```diff
@@ -172,3 +174,23 @@ func (s ServerForClient) GetGameState() gamestate.ClientGameState {
 func (s ServerForClient) GetGameConfig() config.ClientConfig {
 	return s.server.gameConfig.GetClientConfig()
 }
+
+func randomUniqueAssignIIGORoles(input []shared.ClientID) (shared.ClientID, shared.ClientID, shared.ClientID) {
+	lst := make([]shared.ClientID, len(input))
+	copy(lst, input)
+	// Just randomly assign roles if there are not enough clients
```

Comment:
this is fine to me, but you could do this:

```
// shuffle lst
sort.Shuffle(len(lst), func(i, j int) { lst[i], lst[j] = lst[j], lst[i] }
// if >= 3, return the first three
if len(lst) >= 3 {
    return lst[0], lst[1], lst[2]
}
// randomly pick three
return lst[rand.Intn(len(lst))], lst[rand.Intn(len(lst))], lst[rand.Intn(len(lst))]

Reply:
see: https://play.golang.org/p/fBhw9ZTBK9v

Reply:
What do you recommend that I do with the error? createSOMASServer does not return an error. Should I just panic? I was hoping that if there were no clients it would get picked up before this stage.